### ISSUE TITLE
Call Stack window's headers appearing in Debug window

### ISSUE THREAD

**Issue by Robin-Whittle** [eranif/codelite#2321]
I compiled Codelite 13.0.0 from the tar.gz file on Debian 10.  I am accessing it via a Windows 7 machine using Xming.

When stepping through a program (C++, using GDB) and with certain windows open on the lower right: Call Stack, Breakpoint, Threads and Disassembly, the set of bars at the top of the latter window, upon which names of columns are written, also appears at the top of the main Debug window.  In the attached screenshot, it so happens that the program landed on a line near the top of the Debug window, which was redrawn, partially obscuring the errant set of bars.

This is a very low impact bug and perhaps you have already fixed it in the latest source.

BTW, I didn't find a link to this bug tracking system at the main Codelite site, the forum, or the wiki - I thought there was a better place to report bugs than the forum and I found it with Google.

Thanks for Codelite!
![Codelite-UI-bug](https://user-images.githubusercontent.com/13086050/63221633-28546c80-c1df-11e9-9795-6ab98759e039.jpg)

**Comment by Robin-Whittle** [comment]
There is a more disruptive variation on this behaviour.  

I switched from Call Stack to Locals and the currently highlighted line, where execution is currently halted, was overwritten by a grey pattern matching that of the grey bar at the top of the Locals window, with its dividing line between Name and Value.  These words do not appear in the overwritten area.

**Comment by eranif** [comment]
This seems like an artifact caused by the unique setup you are using (xming as your x-server)
I am using Debian on daily basis, and this does not happen on "native" / local xserver
Can you try the same using remote desktop?

**Comment by Robin-Whittle** [comment]
Thanks Eran.  There is no problem when I use the keyboard and screen, and so the X server, of the Debian 10 machine on which Codelite is running, or via sshing to it from another Debian 10 machine.  

With PuTTY > Xming 7.7.0.31 on Windows 7, the problems appear.  Likewise when, on the same Windows 7 machine, I use a program called MobaXTerm, to ssh to the Debian 10 machine and which which can provide an X server in three flavours.  I tried the MobaX, Cygwin_1.14.5 and Cygwin_1.16.3 flavours and they all had the same trouble.  

I also tried a trial version of NetSarang Xmanager-6.0.0080 X server (commercial, closed source) on Windows 7, and the same trouble occurred.  I think this has very different source code from the open source X servers mentioned above, since I found that using it solved some problems I had with the Juce toolkit: http://forum.cabbageaudio.com/t/linux-build-of-cabbage-2-0-02-and-projucer-for-debian-9-x/1031/13 

 With PuTTY > Xming 7.7.0.31 on Windows 7 running Codelite 12.0.11 (which I built from source, using a snapshot .zip file of the source I created from the Git website on 2018-12-27) running on Debian 9.3, there was no problem.

So there is something suspect about the Windows X servers, the Debian version and/or the Codelite version.  The next question is what happens if I compile Codelite 13.0.0 on another Debian 9 machine . . . With Xming the problems DO occur.

So the problem is not between Debian 9 and 10.

The problem occurs with these 4 types of Windows X servers but not a Debian 10 X server.

The problem occurs with Codelite 13.0.0 on Debian 9 and 10, but not with the Codelite 12.0.11 on Debian 9.  (It is not convenient for me to build that old Codelite on a Debian 10 machine now but I will try it when I upgrade one of the Debian 9 machines to 10.)
